docs/tutorials/help-and-placeholder-text-best-practices.md

```diff
@@ -0,0 +1,249 @@
+- Displaying formatted lists of best practices or requirements
+- Including multiple paragraphs of guidance for complex fields
+- Highlighting important warnings or recommendations
+
+### Best Practices for Writing Help Text
+
+1. **Keep it concise**: Aim for 1-2 short sentences that provide immediate clarity
+2. **Include technical constraints**: Mention character limits, formatting requirements, or other constraints
+3. **Explain the purpose**: State why this field matters and how it will be used
+4. **Avoid technical jargon**: Use terminology familiar to your content editors
+5. **Use active voice**: "Enter your headline" is clearer than "Headline should be entered"
+
+> [!TIP]
+> When a field has both character limits and guidance, combine them in the help text rather than relying solely on the UI character counter: "Brief description for social sharing (5-10 words recommended, 30 words maximum)."
+
+## Using Placeholder Text Effectively
+
+Placeholder text appears inside empty fields, suggesting the type of content expected. It disappears when the user begins typing.
+
+> [!NOTE]
+> Placeholders are only supported for specific field types: email, float, integer, string, and url.
+
+### Implementation Example
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  fields: {
+    add: {
+      ctaButtonText: {
+        type: 'string',
+        label: 'myproject:ctaButtonText',
+        help: 'myproject:ctaButtonTextHelp',
+        placeholder: 'myproject:ctaButtonTextPlaceholder'
+      }
+    }
+  }
+};
+```
+<template v-slot:caption>
+  modules/call-to-action/index.js
+</template>
+</AposCodeBlock>
+
+With corresponding translation strings:
+
+<AposCodeBlock>
+
+```json
+{
+  "ctaButtonText": "Button Text",
+  "ctaButtonTextHelp": "The text displayed on the action button.",
+  "ctaButtonTextPlaceholder": "Learn More"
+}
+```
+<template v-slot:caption>
+  modules/myproject/i18n/en/myproject.json
+</template>
+</AposCodeBlock>
+
+### When to Use Placeholder Text
+
+Placeholder text works best in these scenarios:
+- **Showing format examples**: Phone numbers, dates, product codes
+- **Suggesting common values**: "Learn More", "Subscribe", "Download"
+- **Illustrating syntax**: How hashtags or other special formatting should look
+
+### Placeholders and Accessibility
+
+While placeholders can be helpful, they have accessibility limitations:
+- They disappear when typing begins
+- They may have insufficient color contrast
+- Screen readers handle them inconsistently
+
+**For this reason, never use placeholders as the only way to convey essential information.** Always include critical guidance in the help text or label.
+
+## Field Label Writing Tips
+
+Although labels are required for all fields, their quality significantly impacts the editing experience:
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  fields: {
+    add: {
+      // Less helpful label
+      img: {
+        type: 'attachment',
+        label: 'myproject:image'
```

Comment:
The fact that this is just an i18n key and not the actual text makes your message unclear here and spelling it out all the way could be unwieldy too. It might be a rare case where talking about the label text with examples of what the text would be (not code to actually install it) might be more effective than showing a fully clarified configuration example with a JSON file etc. Or should at least appear first so the point can be made cleanly.